index.html

```diff
@@ -81,6 +81,235 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
+                        Multi-Post <br> Stories
+                    </h2>
+                    <h2 class="card-header font-style on-large">
+                        Uber Navigation
+                    </h2>
+                    <section class="card-description">
+                        <ul class="font-style">
+                            <li class="card-list-title on-small"> CANOPY</li>
+                            <li class="card-list-title on-large"> Uber</li>
+                            <li class="Job-title_and_year on-small">
+                                Back End Dev
+                            </li>
+                            <li class="Job-title_and_year on-large">
+                                Lead Developer
+                            </li>
+                            <li class="Job-title_and_year">
+                                2022
+                            </li>
+                        </ul>
+                    </section>
+                    <p class="notice font-style">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="programming-languages font-style">
+                        <li>html</li>
+                        <li class="on-large">Ruby On Rails</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button id ="1" class="project-button button" type="button">See Project</button>
+                </section>
+            </section>
+            <section class="card third-card">
+                <img class="snapshoot-style" src="images/Snapshoot Portfolio3.svg"
+                    alt="Snapshoot portfolio image">
+                <section>
+                    <h1 class="card-header font-style">
+                        Tonic
+                    </h1>
+                    <section class="card-description">
+                        <ul class="font-style">
+                            <li class="card-list-title"> CANOPY</li>
+                            <li class="Job-title_and_year">
+                                Back End Dev
+                            </li>
+                            <li class="Job-title_and_year">
+                                2022
+                            </li>
+                        </ul>
+                    </section>
+                    <p class="notice font-style">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="programming-languages font-style">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button id="2" class="project-button button" type="button">See Project</button>
+                </section>
```

Comment:
- [ ] Kindly fix the design for each project and make sure to match your design with the Figma design and use appropriate padding and margin.

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/212107124-25bcaeae-49aa-414e-b660-e7fba0242ba5.png) |  ![image](https://user-images.githubusercontent.com/83360636/212106634-bdbffd0f-8b3d-4c61-8052-c2f854664912.png)  |